### PR TITLE
Update backend to v3

### PR DESCRIPTION
**Pull request by jimnarey**
Because our routes are now a bit more complex there is no longer such a clear central/non-central government split in terms of what data is collected for a given application. The CentralGovernmentAttributes model no longer makes sense and this removes it.

The application review page will display guidance for reviewers. This creates a model to store guidance for each review section, adds some initial guidance in markdown format and populates the database with it if no existing guidance is found. When we create the new review form we will use the Python-Markdown library to translate the markdown into HTML. This approach makes it easy for superusers to edit the guidance.

Rather than review field by field, we're now going to review in sections which may or may not refer to multiple fields and reviewers will have several choices for each review section, rather than simply approve/reject checkboxes.

### CODE REVIEW

**Review by prakash-gds — CHANGES_REQUESTED** [review]

File: request_a_govuk_domain/request/init_guidance/domain_name_rules.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+How to check:
+
+* Does the domain meet the [.gov.uk naming rules](gov.uk)?
```
Comment (maxf) [inline request_a_govuk_domain/request/init_guidance/domain_name_rules.md:3]: This is probably not the right link

File: request_a_govuk_domain/request/init_guidance/policy_exemption.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+How to check:
+
+* Check that the exemption comes from correct GDS mailbox, and references the correct organisation
+* Check that name of the website requested is aligned with the purpose of the exemption
+* Check directly with the GDS Policy team if necessary - jack.tindale@digital.cabinet-office.gov.uk
```
Comment (maxf) [inline request_a_govuk_domain/request/init_guidance/policy_exemption.md:5]: Isn't there a more generic mailbox?

File: request_a_govuk_domain/request/init_guidance/registrant_permission.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+How to check:
+
+* Check that the person giving the permission is identified by name and identifies themselves as a C-suite officer (CIO, CTO, CEO, CFO, CISO etc)
```
Comment (maxf) [inline request_a_govuk_domain/request/init_guidance/registrant_permission.md:3]: Abbreviations are supported in python-markdown. See https://python-markdown.github.io/extensions/abbreviations/

File: request_a_govuk_domain/request/init_guidance/registrant_person.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+* For Parish Council Clerks
+    * Check any existing websites that belong to the organisation for the named person and their role as Parish Clerk
+    * Check the websites of the Principal Authorities that cover the Parish for the named person and their role as Parish Clerk
+    * Enquire with NALC and/or SLCC to confirm whether they are the recognised Parish Clerk
+* For all others:
+    * Check WHOIS records to see whether the Registrant’s email address belongs to the organisation
+    * Check any existing websites that belong to the organisation for the named person and their role
+    * For new organisations, check WHOIS records to see whether the Registrant’s email address belongs to an existing organisation that we know is setting up the new organisation
+    * Any evidence online of the Registrant conducting any activity on behalf of the organisation.”
```
Comment (maxf) [inline request_a_govuk_domain/request/init_guidance/registrant_person.md:11]: trailing " ?

**Review by maxf — CHANGES_REQUESTED** [review]

File: request_a_govuk_domain/request/management/commands/create_sample_data.py [hunk 5]
```diff
@@ -36,46 +98,48 @@ def handle(self, *args, **options):
         models.RegistrantPerson.objects.all().delete()
         models.Registrant.objects.all().delete()
         models.Application.objects.all().delete()
-        models.CentralGovernmentAttributes.objects.all().delete()
         models.Review.objects.all().delete()
 
-        registrant_person = models.RegistrantPerson.objects.create(name=PERSON_NAMES[0])
+        # Handle Registrars differently, as we might have already loaded the list of real registrars
+        # before this is run
+        if not models.Registrar.objects.all():
```
Comment (prakash-gds) [inline request_a_govuk_domain/request/management/commands/create_sample_data.py:105]: We need DUMMY_REGISTRAR in local testing

**Review by prakash-gds — APPROVED** [review]

**Review by maxf — APPROVED** [review]

**Pull request merged**